index.js

```diff
@@ -7,6 +7,20 @@ const crypto = require('crypto');
 const stringify = require('json-stable-stringify');
 const stripBom = require('strip-bom');
 
+function rethrowBuildError(error) {
+  if (!error) { throw new Error('Unknown Error'); }
```

Comment:
i believe we can still share `realtivePath` in some useful way if we throw here. RIght?